cmd/zap/cmd/vector.go

```diff
@@ -139,32 +138,12 @@ func decodeSection(data []byte, start uint64) (int, int, map[int64][]uint32, *fa
 	numVecs, n := binary.Uvarint(data[pos : pos+binary.MaxVarintLen64])
 	pos += n
 	for i := 0; i < int(numVecs); i++ {
-		vecID, n := binary.Varint(data[pos : pos+binary.MaxVarintLen64])
+		vecID, n := binary.Uvarint(data[pos : pos+binary.MaxVarintLen64])
```

Comment:
this should be binary.Varint() right? like i see that in other parts of the code the vecID is retained as binary.Varint()

Reply:
yeah, my bad - accidentally changed this for running some edge case tests.